src/components/forms/radio-button-group/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,49 @@
+/** @jsxImportSource @emotion/react */
+import { RadioGroup, type RadioGroupProps } from '@blueprintjs/core';
+import { css } from '@emotion/react';
+
+import { type InputVariant } from '../styles.js';
+
+import { RadioButtonItem } from './ButtonRadioItem.js';
+
+export interface RadioButtonGroupProps extends RadioGroupProps {
+  variant?: InputVariant;
```

Comment:
Do I keep this `variant` prop? I think it's better to use `large` prop similar to blueprintjs components

Reply:
I agree, ok to change for a `large` boolean prop.